src/view/editor/parts/editor-time-zone.tsx

```diff
@@ -34,7 +34,7 @@ export const EditorTimezone = ({ editorId }: { editorId: string }): ReactElement
 
 	const { zimbraPrefUseTimeZoneListInCalendar } = usePrefs();
 	const timeZonesOptions = useMemo(() => TimeZonesOptions(t), [t]);
-
+	const timezoneLabel = t('timezones', 'Timezones');
 	const [value, setValue] = useState<SelectValue>(() => {
 		if (timezone && zimbraPrefUseTimeZoneListInCalendar === 'TRUE') {
```

Comment:
I suggest to rename the variable in order to remove the Zimbra name as much as possible

Reply:
I will create a custom hook which will be responsible of exporting renamed prefs